standalone/pruntime/enclave/src/system/gk.rs

```diff
@@ -178,11 +179,13 @@ where
     MsgChan: MessageChannel,
 {
     fn process(&mut self) {
+        debug!("Gatekeeper: processing block {}", self.block.block_number);
```

Comment:
Do you think it's better to assign a specific target to these logs so that we can control the output by RUST_LOG?

Reply:
The default target is the module, so we can control it by RUST_LOG="enclaveapp::system::gk=debug".
I think it's enough for now.